tests/CMakeLists.txt

```diff
@@ -29,18 +29,26 @@ else()
     set(TEST_RUNNER "")
 endif()
 
+
 add_library(rascal_tests_main STATIC helpers/tests_main.cc)
+# make sure all tests include the boost that has been found
+target_include_directories(rascal_tests_main SYSTEM PRIVATE ${Boost_INCLUDE_DIR})
```

Comment:
If you set this as PUBLIC instead of PRIVATE, then you don't have to set it again below

Reply:
I feel it is more explicit to add the includes to all test binaries so that it is more explicit.

Reply:
It is more explicit, but it also goes against the usual "modern" cmake way of doing this. The whole point of targets & targets properties is to remove the repetition of include/compiler flags required.